xbmc/input/InputManager.h

```diff
@@ -230,7 +230,19 @@ class CInputManager : public ISettingCallback
   void RegisterKeyboardHandler(KEYBOARD::IKeyboardHandler* handler);
   void UnregisterKeyboardHandler(KEYBOARD::IKeyboardHandler* handler);
 
-  void RegisterMouseHandler(MOUSE::IMouseInputHandler* handler);
+  /*! \brief Registers a handler to be called on mouse input (e.g a game client).
+   *
+   * \param handler The handler to call on mouse input.
+   * \return[in] The controller ID of the button map used to translate from windowing
```

Comment:
I would say `\return The controller ID that serves as a context for incoming events`